modules/vertice-governance-role/variables.tf

```diff
@@ -10,6 +10,12 @@ variable "cur_bucket_name" {
   default     = null
 }
 
+variable "governance_role_external_id" {
+  type        = string
+  description = "STS external ID value to require for assuming the governance role. You will receive this from Vertice. Required if governance role is to be created."
+  default     = ""
```

Comment:
Nevermind, discussed offline. The solution is good as it is cause role is not required so its parameters also should not be.

Reply:
We only want to require the external ID in case `governance_role_enabled` is true, and Terraform doesn't allow referencing other variables in the `validation` block.